root/static/scripts/edit/MB/TextList.js

```diff
@@ -35,11 +38,17 @@ MB.Form.TextList = function (input) {
     self.add = function (init_value) {
         $template.clone()
             .removeClass(template)
-            .insertAfter($template.parent().find('div.text-list-row').last())
+            .insertAfter($template
+            .parent()
+            .find('div.text-list-row')
+            .last())
```

Comment:
Unlike the ones further down (which do seem right to indent together with the rest as @mwiencek said) these ones here apply to $template, not .insertAfter, so I'd expect them to be indented one more step.